src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const lines = sourceString.split(';');
```

Comment:
When splitting the `sourceString` by semicolons, you may end up with empty strings in the `lines` array if there are multiple semicolons with no styles between them. While it's not required to handle this case, it would be more robust to filter out empty strings.